backend/src/test/java/com/festago/application/integration/TicketServiceIntegrationTest.java

```diff
@@ -72,59 +66,59 @@ class TicketServiceIntegrationTest extends ApplicationIntegrationTest {
-            .existsByOwnerAndStage(any(Member.class), any(Stage.class));
+        LocalDateTime stageStartTime = LocalDateTime.parse("2022-07-26T18:00:00");
+        doReturn(stageStartTime.minusWeeks(1).toInstant(ZoneOffset.UTC))
+            .when(clock)
+            .instant();
+        Festival festival = festivalRepository.save(FestivalFixture.festival()
+            .startDate(stageStartTime.toLocalDate())
+            .endDate(stageStartTime.toLocalDate())
+            .build());
+        Stage stage = stageRepository.save(StageFixture.stage()
+            .festival(festival)
+            .startTime(stageStartTime)
+            .ticketOpenTime(stageStartTime.minusDays(1))
+            .build());
+        TicketCreateRequest request = new TicketCreateRequest(stage.getId(), TicketType.VISITOR,
+            100, stageStartTime.minusHours(1));
 
         // when
-        List<CompletableFuture<Void>> futures = IntStream.range(0, tryCount)
-            .mapToObj(i -> CompletableFuture.runAsync(() -> {
-                ticketService.ticketing(member.getId(), request);
-            }, executor).exceptionally(e -> null))
-            .toList();
-        futures.forEach(CompletableFuture::join);
+        TicketCreateResponse response = ticketService.create(request);
 
         // then
-        assertThat(memberTicketRepository.count()).isEqualTo(50);
+        TicketAmount ticketAmount = ticketAmountRepository.findById(response.id()).get();
+        assertThat(ticketAmount.getTotalAmount()).isEqualTo(100);
     }
 
     @Test
-    @Sql("/ticketing-test-data.sql")
-    void 중복으로_티켓을_예매하면_예외() {
+    void 티켓이_있는_공연에_티켓을_추가하면_기존_티켓의_수량이_증가() {
         // given
-        Member member = memberRepository.save(MemberFixture.member().build());
-        TicketingRequest request = new TicketingRequest(1L);
-        Long memberId = member.getId();
-        ticketService.ticketing(memberId, request);
-
-        // when & then
-        assertThatThrownBy(() -> ticketService.ticketing(memberId, request))
-            .isInstanceOf(BadRequestException.class)
-            .hasMessage("예매 가능한 수량을 초과했습니다.");
-    }
+        LocalDateTime stageStartTime = LocalDateTime.parse("2022-07-26T18:00:00");
+        doReturn(stageStartTime.minusWeeks(1).toInstant(ZoneOffset.UTC))
+            .when(clock)
+            .instant();
+        Festival festival = festivalRepository.save(FestivalFixture.festival()
+            .startDate(stageStartTime.toLocalDate())
+            .endDate(stageStartTime.toLocalDate())
+            .build());
+        Stage stage = stageRepository.save(StageFixture.stage()
+            .festival(festival)
+            .startTime(stageStartTime)
+            .ticketOpenTime(stageStartTime.minusDays(1))
+            .build());
+        TicketCreateRequest request = new TicketCreateRequest(stage.getId(), TicketType.VISITOR,
+            100, stageStartTime.minusHours(1));
+
+        ticketService.create(request);
```

Comment:
도메인 특성상 given절이 길어지는건 어쩔수 없는 부분이지만 메소드 추출을 통해 가독성을 개선할 수 있을 것 같습니다. !!
예를 들어서..
```suggestion
        // given
        LocalDateTime stageStartTime = LocalDateTime.parse("2022-07-26T18:00:00");
        현재시간은_다음과_같다(stageStartTime.minusWeeks(1));
        Festival festival = 축제를_생성한다(stageStartTime.toLocalDate());
        Stage stage = 공연을_생성한다(stageStartTime, festival);
        티켓을_생성한다(stage.getId(), TicketType.VISITOR, 100, stageStartTime.minusHours(1));
```